projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -41,12 +41,21 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
     /**
     * Provides an observable collection of all `IgxTabsGroupComponent`s.
     * ```typescript
-    * const groupItems = this.myTabComponent.tabs;
+    * const groupItems = this.myTabComponent.groups;
     * ```
     */
     @ContentChildren(forwardRef(() => IgxTabsGroupComponent))
     public groups: QueryList<IgxTabsGroupComponent>;
 
+    /**
+    * Provides an observable collection of all `IgxTabItemComponent`s defined in the page.
+    * ```typescript
+    * const tabItems = this.myTabComponent.tabsAsContentChildren;
+    * ```
+    */
+    @ContentChildren(forwardRef(() => IgxTabItemComponent))
+    public contentTabs: QueryList<IgxTabItemComponent>;
```

Comment:
Make the tabs property to return either the contentTabs or the viewTabs.

Reply:
Addressed.